VideoPlayer/tests/VideoPlayer-specs.js

```diff
@@ -0,0 +1,164 @@
+import '@testing-library/jest-dom';
+import {act, createEvent, fireEvent, render, screen, waitFor} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import VideoPlayer from '../VideoPlayer';
+
+const focus = (slider) => fireEvent.focus(slider);
+const keyDown = (keyCode) => (button) => fireEvent.keyDown(button, {keyCode});
+const downKeyDown = keyDown(40);
+
+describe('VideoPlayer', () => {
+	test('should fire `onPlaying` with `playing` type when playing event is fired', () => {
+		const handlePlaying = jest.fn();
+
+		render(
+			<VideoPlayer data-testid="videoplayer-id" onPlaying={handlePlaying} />
+		);
+
+		const video = screen.getByTestId('videoplayer-id');
+
+		const playingEvent = createEvent('playing', video);
+		fireEvent(video, playingEvent);
+
+		const expected = {type: 'playing'};
+		const actual = handlePlaying.mock.calls.length && handlePlaying.mock.calls[0][0];
+
+		expect(actual).toMatchObject(expected);
+	});
+	test('should fire `onControlsAvailable` with `onControlsAvailable` type when screen clicked', () => {
```

Comment:
Please add an empty line between each test.

Reply:
Added it